lessons/git/ci-github/lesson.md

```diff
@@ -113,6 +156,37 @@ I am not nearly as familiar with Python, however, the same things that apply to
 also apply to Python. You (optionally) need to install dependencies and use the `script` option
 to specify what commands or file to run the tests with.
 
+As with R, let's do a simple and silly example. Create the three files: `.travis.yml`, 
+`function.py`, and `test.py`. (As a side note, as with above, I will not be covering 
+formal unit testing in this lesson).
+
+**`.travis.yml`**
+
+```
+language: python
+script:
+    - python test.py
+```
+
+**`function.py`**
+
+```r
+def add_nums(num1, num2):
+    added = num1 + num2
+    return added;
+```
+
+**`test.py`**
+
+```r
+from function import *
+
+added = add_nums(2, 2)
```

Comment:
For the whole file:
```python
import function

def test_add_nums():
    added = function.add_nums(2, 2)
    assert added == 4
```
Since this isn't about unit testing, I won't go into anything like parametrization or the like.